types/sinon/ts3.1/index.d.ts

```diff
@@ -334,7 +334,7 @@ declare namespace Sinon {
          */
         restore(): void;
         /**
-         * Get the original method. Only available if the spy replaced an existing method.
+         * Get the original method. Only available if the spy replaced an existing method, but not a property.
```

Comment:
are you sure this is true? if you spy on an accessor (i.e a property), the wrapped accessor would have a wrappedMethod wouldn't it?

as in:

```
const spy = sinon.spy(obj, 'prop', ['get']);
spy.get.wrappedMethod;
```